test/solc/CommandLineInterface.cpp

```diff
@@ -1565,213 +1532,162 @@ BOOST_AUTO_TEST_CASE(cli_ethdebug_debug_info_ethdebug)
 			{"solc",  "--ethdebug-runtime", "--strict-assembly", tempDir.path().string() + "/input.yul"},
-			{"Error: The following outputs are not supported in assembler mode: --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc",  "--ethdebug", "--ethdebug-runtime", "--strict-assembly", tempDir.path().string() + "/input.yul"},
-			{"Error: The following outputs are not supported in assembler mode: --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "ethdebug", "--ethdebug", tempDir.path().string() + "/input.sol"},
-			{"Error: --ethdebug / --ethdebug-runtime output can only be selected, if --via-ir was specified.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "ethdebug", "--ethdebug-runtime", tempDir.path().string() + "/input.sol"},
-			{"Error: --ethdebug / --ethdebug-runtime output can only be selected, if --via-ir was specified.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "ethdebug", "--ethdebug", "--ethdebug-runtime", tempDir.path().string() + "/input.sol"},
-			{"Error: --ethdebug / --ethdebug-runtime output can only be selected, if --via-ir was specified.\n"},
-			{},
-		},
-		{
-			{"solc", "--debug-info", "ethdebug", "--ethdebug", "--ethdebug-runtime", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{},
-			{"======= Debug Data (ethdebug/format/info/resources) =======", "Debug Data (ethdebug/format/program):", "Debug Data of the runtime part (ethdebug/format/program):"},
 		},
 		{
 			{"solc", "--debug-info", "location", "--ethdebug", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{"Error: --debug-info must contain ethdebug, when compiling with --ethdebug / --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "location", "--ethdebug-runtime", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{"Error: --debug-info must contain ethdebug, when compiling with --ethdebug / --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "location", "--ethdebug", "--ethdebug-runtime", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{"Error: --debug-info must contain ethdebug, when compiling with --ethdebug / --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "all", "--ethdebug", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{"Error: --debug-info must contain ethdebug, when compiling with --ethdebug / --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "all", "--ethdebug-runtime", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{"Error: --debug-info must contain ethdebug, when compiling with --ethdebug / --ethdebug-runtime.\n"},
-			{},
 		},
 		{
 			{"solc", "--debug-info", "all", "--ethdebug", "--ethdebug-runtime", "--via-ir", tempDir.path().string() + "/input.sol"},
-			{"Error: --debug-info must contain ethdebug, when compiling with --ethdebug / --ethdebug-runtime.\n"},
-			{},
 		},
 	};
-	for (auto const& test: tests)
+	static std::vector<std::vector<std::string>> successful_tests{
```

Comment:
This should be done with data test cases and split into two, imo.

```suggestion
	static std::vector<std::vector<std::string>> supportedCLIFlagCombinations{
```